package.json

```diff
@@ -1,23 +1,26 @@
 {
-  "name": "javascript-nodejs-starter",
+  "name": "nodejs-website",
   "version": "2.0.0",
   "private": true,
   "scripts": {
     "start": "node app.js"
   },
   "dependencies": {
     "body-parser": "~1.0.0",
+    "cookies": "^0.7.0",
     "debug": "~0.7.4",
     "errorhandler": "~1.1.1",
     "express": "~4.13.4",
     "method-override": "~2.1.1",
     "morgan": "~1.0.0",
+    "prismic-dom": "^1.2.8",
```

Comment:
it's not the latest version.
idem for prismic-javascript.
prismic-nodejs needs to be removed.
Can you update all dependencies to the last version available and make sure it's still working, it's a good opportunity to update everything.